src/index.js

```diff
@@ -1,17 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
There is no need of importing the react since you are using react version 18.2, react handles it for you, and you don't need to import the react instead you can name import the component of StricMode only.
```suggestion
import {StrictMode} from "react";
```